libs/portal-integration-angular/src/lib/core/components/button-dialog/button-dialog.component.html

```diff
@@ -7,7 +7,7 @@
             pButton 
             *ngIf="dialogData.config.secondaryButtonIncluded"
             class="mx-2"
-            [icon]="dialogData.config.secondaryButtonDetails!.icon !== undefined ? dialogData.config.secondaryButtonDetails!.icon : ''"
+            icon="{{dialogData.config.secondaryButtonDetails!.icon !== undefined ? dialogData.config.secondaryButtonDetails!.icon : ''}}"
```

Comment:
Why is this needed?

Reply:
With the previous implementation typing error occurs.
`Type 'PrimeIcons' is not assignable to type 'string'`